Dockerfile

```diff
@@ -4,7 +4,8 @@ MAINTAINER Michael J. Stealey <stealey@renci.org>
 ### Begin - HydroShare Development Image Additions ###
 RUN pip install --upgrade pip && pip install \
   robot_detection \
-  django-ipware
+  django-ipware \
```

Comment:
@alexlemann - developer level changes will start out in the local Dockerfile so that they can be picked up by all once merged. Prior to deployment a new hs_docker_base image will be generated to include all accumulated Dockerfile changes which do get put into the requirements.txt file. This is done to prevent the developer from having to download a ~3 GB image file each time a small change is made, or get out of step with the current base image.